### ISSUE TITLE
Move Planner

### ISSUE THREAD

**Issue by DeamonHunter** [DeamonHunter/AWBW-Replay-Player#62]
Would probably require a good amount of work in order to allow this to happen.

**Comment by EinsameLyc** [comment]
Yes, planner is a very useful feature. It seems that planner on the web is occasionally buggy, it would be great if planner could be applied on the Player. Looking forward to it!
